src/wovnio/utils/request_handlers/FileGetContentsRequestHandler.php

```diff
@@ -36,11 +37,15 @@ protected function post($url, $request_headers, $data, $timeout)
         list($response, $response_headers) = $this->fileGetContents($url, $http_context);
 
         if ($response === false) {
-            preg_match('{HTTP\/\S*\s(\d{3})}', $response_headers[0], $match);
+            $error_type = '';
+            if ($last_error = error_get_last()) {
+                $error_type = $last_error['type'];
+                Logger::get()->error($last_error['message']);
+            }
 
-            $http_error_code = $match[1];
+            $error_in_response = $response_headers[0];
```

Comment:
I googled a bit and it seems that `$http_response_header` can be unset in rare cases (in the case of HTTP 100 for example). So an `@` to suppress any potential warning message would be good.